src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+fieldset {
```

Comment:
use classes as selectors

Reply:
everything is fixed. Are there any other comments?